ONSdigital/eq-terraform
# Eq 31 smoke test

What
Smoke test added to terraform scripts

How to test
1. Make sure [smoke test ](https://github.com/ONSdigital/eq-smoke-test/pull/2) has been merged
2. Run terraform scripts terraform apply --var "env=test"
3. Make sure there are no failures

Who can test
Anyone but @LJBabbage
